nvflare/client/ex_process/api.py

```diff
@@ -93,7 +93,8 @@ def init(self, rank: Optional[str] = None):
         flare_agent = None
         try:
             if rank == "0":
-                if client_config.get_exchange_format() == ExchangeFormat.PYTORCH:
+                if client_config.get_exchange_format() in [ExchangeFormat.PYTORCH, ExchangeFormat.NUMPY]:
+                    # both numpy and pytorch exchange format can need tensor decomposer
```

Comment:
numpy exchange format shouldn't need TensorDecomposer. Something is off.

Reply:
makes sense, will figure out a better way to address the issue

Reply:
Tried to find alternatives, but seems it has to be loaded at this API level, otherwise the external process cannot find it. TensorDecomposer is needed by PT when exchange with numpy - PT can have two exchange modes: tensor or numpy, so numpy exchange form can still need TensorDecomposer (to convert the received numpy to tensor that can be loaded).

Reply:
To make it clearer, we may need to further decouple "framework" (TF, PT, Numpy, etc.) and "exchange format", one framework can use multiple exchange format, but that needs a more thorough design to make sure nothing's broken.

Reply:
I removed the fobs.register at API, perform import only. register will be performed by the particular framework code when needed (actually we previously "double register" for TensorDecomposer here https://github.com/NVIDIA/NVFlare/blob/main/nvflare/app_opt/pt/client_api_launcher_executor.py#L28 )